src/main/java/Dan/TaskList.java

```diff
@@ -0,0 +1,103 @@
+package Dan;
+
+import java.util.List;
+import java.util.ArrayList;
+
+public class TaskList {
+    private List<Task> tasks;
+
+    public TaskList(List<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public void addTask(String input) throws DanException {
+        String description;
+        String dateString;
```

Comment:
Did you forget access modifiers here?

Reply:
These are actually part of addTask method, and not an attribute of the class